build/common.ps1

```diff
@@ -181,7 +181,7 @@ Function Install-NuGet {
     )
     if ($Force -or -not (Test-Path $NuGetExe)) {
         Trace-Log 'Downloading nuget.exe'
-        wget https://dist.nuget.org/win-x86-commandline/latest-prerelease/nuget.exe -OutFile $NuGetExe
+        wget https://dist.nuget.org/win-x86-commandline/v3.5.0/nuget.exe -OutFile $NuGetExe
```

Comment:
Seems like having a local build and CI build diverge (more) is a bad thing, right?